app/assets/javascripts/channels/room.coffee

```diff
@@ -12,10 +12,17 @@ App.room = App.cable.subscriptions.create "RoomChannel",
     $("#game_cards").append data["game_card"]
     $("#pot").text data["pot"]
     $("##{data["turn"]}").addClass("highlight")
+
     if data["start_game"]
+      console.log("start")
```

Comment:
Unless you have a really good reason for it, I would avoid leaving console.log's in your master branch.